packages/twenty-front/src/modules/settings/components/SettingsRadioCard.tsx

```diff
@@ -34,25 +34,39 @@ const StyledDescription = styled.div`
 
 type SettingsRadioCardProps = {
   value: string;
-  handleClick: (value: string) => void;
+  handleSelect: (value: string) => void;
   isSelected: boolean;
   title: string;
   description?: string;
   Icon?: IconComponent;
+  role?: string;
+  ariaChecked?: boolean;
 };
 
 export const SettingsRadioCard = ({
   value,
-  handleClick,
+  handleSelect,
   title,
   description,
   isSelected,
   Icon,
 }: SettingsRadioCardProps) => {
   const theme = useTheme();
 
+  const onClick = () => handleSelect(value);
+  const onKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter' || e.key === ' ' || e.code === 'Space') {
+      e.preventDefault();
+      handleSelect(value);
+    }
+  };
```

Comment:
Ideally we should use hotkey scopes for keyboard management.

An ideal implementation of a radio group would switch option when clicking on down and up arrows.

Reply:
You're right I will do it. 

But we should use an input radio which natively integrates all features to deal with it. Here we need to fake the usage. It will be hard to maintain. 

IMO the radio button should be refactored.